zap/src/main/java/org/parosproxy/paros/extension/ViewDelegate.java

```diff
@@ -146,4 +153,46 @@ default KeyStroke getMenuShortcutKeyStroke(int keyCode, int modifiers, boolean o
                 Toolkit.getDefaultToolkit().getMenuShortcutKeyMask() | modifiers,
                 onKeyRelease);
     }
+
+    /**
+     * Gets an options button, accepting a tooltip and options panel name (to be displayed when
+     * opening the options dialog).
+     *
+     * <p>Should be called/used only when in view mode.
+     *
+     * @param tooltip the text to be displayed in the button's tooltip.
+     * @param optionsPanelName the name of the panel to be displayed when the Options dialog panel
+     *     is opened.
+     * @return the options button, never {@code null}.
+     * @since 2.13.0
+     */
+    static JButton getOptionsButton(String tooltip, String optionsPanelName) {
```

Comment:
I'd go with `default` (I know I said static earlier but that was if they were kept in the extensions), to allow the view implementation to override the behaviour.